packages/ast/src/index.js

```diff
@@ -251,13 +251,15 @@ export function blockInstruction(
   assert(typeof label.type === "string");
   assert(typeof instr === "object" && typeof instr.length !== "undefined");
 
-  return {
+  const node: BlockInstruction = {
     type: "BlockInstruction",
     id: "block",
     label,
     instr,
     result
   };
+
+  return node;
```

Comment:
The return type is already `BlockInstruction`, didn't flow catch that?

Reply:
Good spot - that is a hangover from a refactor / revert